stone/target/python_type_mapping.py

```diff
@@ -0,0 +1,88 @@
+from typing import (
```

Comment:
Alternatively I could just do `import typing` and just refer to them as `typing.Callable`